test/powershell/Modules/Microsoft.PowerShell.Management/Get-Location.Tests.ps1

```diff
@@ -2,20 +2,80 @@
 # Licensed under the MIT License.
 Describe "Get-Location" -Tags "CI" {
     $currentDirectory=[System.IO.Directory]::GetCurrentDirectory()
+
     BeforeEach {
-	Push-Location $currentDirectory
+        Push-Location $currentDirectory
     }
 
     AfterEach {
-	Pop-location
+	    Pop-location
```

Comment:
Indentation.
We use 4 spaces (not tabs) per indentation step. Please fix below too.